### DISCUSSION TITLE
Inconsistency in documentation for Release Channels

### DISCUSSION THREAD

**Discussion by cbuchmannNM** [rancher/rke2#2827] [Q&A] [answered]
I think it's just a oversight in the documentation;
Under install docs are multiple release channels mentioned:  
`Channel to use for fetching RKE2 download URL. Defaults to stable. Options include: stable, latest, testing.`

But in the Upgrade Basics it is mentioned that only the channel "latest" exists: 
`Currently, the latest channel is the only available channel. Once we have more releases and need to distinguish between the most recent release and the most stable release, we will add a stable channel and set it as the default.`

Checking the [release channels manually](https://update.rke2.io/v1-release/channels) I can see multiple channels.

Am I missing something regarding this topic?
Thank you in advance.

**Comment by brandond** [comment, marked as answer]
nope, just stale docs. There are also channels for each minor version, same as for K3s.
